crates/host/src/wasmbus/mod.rs

```diff
@@ -1649,80 +1717,62 @@ impl Host {
                     )
                     .await?;
 
-                if let Err(e) = self
-                    .start_component(
-                        entry,
-                        wasm,
-                        claims.clone(),
-                        Arc::clone(&component_ref),
-                        Arc::clone(&component_id),
-                        max,
-                        annotations.clone(),
-                        config,
-                        secrets,
-                    )
-                    .await
-                {
-                    self.publish_event(
-                        "component_scale_failed",
-                        event::component_scale_failed(
-                            claims.as_ref(),
-                            &annotations,
-                            host_id,
-                            &component_ref,
-                            &component_id,
-                            max,
-                            &e,
-                        ),
-                    )
-                    .await?;
-                    return Err(e);
-                }
+                self.start_component(
+                    entry,
+                    wasm,
+                    claims.clone(),
+                    Arc::clone(&component_ref),
+                    Arc::clone(&component_id),
+                    max,
+                    annotations,
+                    config,
+                    secrets,
+                )
+                .await?;
```

Comment:
```suggestion
                .await.context("failed to start component while processing scale request")?;
```

Reply:
Will grab this in a followup to keep the review simple!